### PR TITLE
4.6 release notes: Kuryr NetlinkError known issue

### PR DESCRIPTION
**Pull request by dulek**

### CODE REVIEW

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -2114,6 +2114,13 @@ Automatic port deletion is planned for a future release of
 {product-title}.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1888318[*BZ#1888318*])
 
+* It's known that deployments on {rh-openstack} configured with Kuryr may experience
+kuryr-cni pods getting into a CrashLoop due to `NetlinkError: (17, 'File exists')`
+If that happens a workaround is to reboot such a node. This is planned to be
+fixed in a future release of {product-title}.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1869606[*BZ#1869606*])
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:2122]:
```suggestion
* Deployments on {rh-openstack} configured with Kuryr might experience kuryr-cni pods going into a CrashLoop, which reports a `NetlinkError: (17, 'File exists')` error. As a workaround, you must reboot the node. A fix is planned to resolve this issue in a future release of {product-title}. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1869606[*BZ#1869606*])
```

**Review by codyhoag — COMMENTED** [review]
@dulek can you verify that my suggested change makes sense? Thanks!

**Comment by dulek** [comment]
@codyhoag: Changes make total sense, thanks!

**Comment by codyhoag** [comment]
@dulek If you can squash the commits, I will merge.

**Comment by dulek** [comment]
@codyhoag: Done!

**Review by vikram-redhat — APPROVED** [review]

**Comment by codyhoag** [comment]
Thanks @dulek !

**Pull request merged**